### ISSUE TITLE
Could not open model_speech/speech_model251_e_0_step_625000.model: Data loss: not an sstable (bad magic number): perhaps your file is in a different file format and you need to use a different restore operator?

### ISSUE THREAD

**Issue by bibo19842003** [nl8590687/ASRT_SpeechRecognition#221]
版本：ASRT_v0.6.1

执行： python3 asrserver.py  后有如下错误输出：
Could not open model_speech/speech_model251_e_0_step_625000.model: Data loss: not an sstable (bad magic number): perhaps your file is in a different file format and you need to use a different restore operator?

**Comment by nl8590687** [comment]
可能是运行环境没安装对，请按照教程来试试
https://blog.ailemon.me/2020/08/27/teach-you-how-use-asrt-deploy-chinese-asr-api-server/

**Comment by james-space** [comment]
@bibo19842003 我也遇到了一样的问题，应该是tensorflow和keras版本的关系（tensorflow  2.3.1，Keras 2.4.3），我退到博主的版本后可以正常运行（tensorflow  1.15.0，Keras 2.3.1）。

**Comment by bibo19842003** [comment]
@nl8590687 @james-space 
感谢二位的帮忙
